### ISSUE TITLE
Comments on the terminology used

### ISSUE THREAD

**Issue by mdmk** [cairoshell/cairoshell#89]
I think some words should be more precise.
So I'll list them and put my suggestion and explain why.

Quick Launch --> Quick Launch Area
Desktop Overlay --> Dynamic Desktop Overlay
Toolbar --> Desktop Navigation Bar

Speaking of quick lauch makes me think of autostart or something, an action, not the place I would seek for icons.
Desktop overlay makes me think of an overlay, like FPS counter, that would display on top of the desktop, With this name, I don't expect the desktop itself to be the overlay (at least I would not if I did not knew the feature).
Toolbar is too generic, as it does not contain any tool, but is used for navigation, exclusively for the desktop.

**Comment by dremin** [comment]
I agree on all of your points and will implement these changes, thanks for the feedback.

**Comment by mdmk** [comment]
Some more wording suggestions.

When you right click on a quick lauch icon, "Remove from menu" is inacurate as pointed out before
"Remove" would be sufficient, like it is done to unpin special folders (Documents, Download, etc.) on menu bar.
Or "Unpin" (?), to stick to the native Windows words.

If you remove Quick Lauch app, there is "Remove this app from the menu?" and ""<app>" will be removed from the Quick Lauch menu. This will not uninstall the program."
Same remarks as before concerning "menu" as not accurate.

The word "Special Folders" is generic.
The word "Places" is already used to group the same "Special Folders" in one menu on the menu bar.
So, maybe reuse this word ?
It is similar to Recent Places (?) so not so bad.
In Windows 7, there were a word for special folders, it's "Libraries", but this concept is disabled by default in Windows 10, not a good idea.

**Comment by dremin** [comment]
@mdmk Thanks, I've made some more updates with your comments in mind.
